core/tests/webdriverio_utils/TopicAndStoryViewerPage.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the topic and story viewer page, for use
+ * in WebdriverIO tests.
+ */
+
+var action = require('../webdriverio_utils/action.js');
+var waitFor = require('./waitFor.js');
+
+var TopicAndStoryViewerPage = function() {
+  var lessonTrack = $('.e2e-test-lesson-track');
+  var practiceSessionContainer = $('.e2e-test-practice-session-container');
+
+  this.get = async function(
+      classroomUrlFragment, topicUrlFragment, storyUrlFragment) {
+    await browser.url(
+      `/learn/${classroomUrlFragment}/${topicUrlFragment}` +
+      `/story/${storyUrlFragment}`);
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.goToChapterIndex = async function(index) {
+    var chapterTitleList = await $$('.e2e-test-chapter-title');
```

Comment:
Earlier declared at the top. Ditto elsewhere.

Reply:
Done.